NitroxClient/GameLogic/ItemContainers.cs

```diff
@@ -89,5 +108,28 @@ public void RemoveItem(string ownerGuid, string itemGuid)
                 Log.Error("Could not find container field on object " + owner.name);
             }
         }
+
+        public string GetLockerGuid(Transform ownerTransform)
+        {
+            string LockerId = ownerTransform.gameObject.name.Substring(7, 1);
+
+            GameObject locker = ownerTransform.parent.gameObject.FindChild("submarine_locker_01_0" + LockerId);
+            if (locker != null)
+            {
+                foreach (Transform tr in locker.transform)
+                {
+                    foreach (Transform tr2 in tr)
+                    {
+                        StorageContainer storageContainer = tr2.GetComponent<StorageContainer>();
+                        if (storageContainer != null)
+                        {
+                            return GuidHelper.GetGuid(storageContainer.gameObject);
+                        }
+                    }
+                }
+            }
+
+            return string.Empty;
```

Comment:
We should throw an exception so that this doesn't become a silent failure if UWE changes the tree structure.